addon/components/bootstrap-inputs/-text.js

```diff
@@ -0,0 +1,93 @@
+import Ember from 'ember';
+import layout from '../../templates/components/bootstrap-inputs/-text';
+import { PropTypes } from 'ember-prop-types';
+import { BuilderForPropTypes, BuilderForPropDefaults } from 'ember-bootstrap-controls/utils/prop-definition-tools';
+
+const autocompleteTypes = ['off', 'on', 'name', 'honorific-prefix',
```

Comment:
We should remove all of the `tel` related ones because we have https://github.com/wildland/ember-bootstrap-controls/pull/172 that is better for telephone inputs.